README.md

```diff
@@ -210,29 +211,40 @@ open docs/_build/html/index.html
+If you’d like to receive Slack notifications whenever a new release is published, follow the comments in the [Release Notification](https://github.com/jenstroeger/python-package-template/tree/main/.github/workflows/release-notifications.yaml) Action and set up a Slack bot by following [the instructions here](https://github.com/slackapi/slack-github-action#setup-2).
 
-You can also install and run the tool manually and locally, for example:
+In order to build a distribution of your package locally instead of publishing it through the Github Actions workflow, you can simply call:
 
 ```bash
-pip install python-semantic-release
-semantic-release changelog
-semantic-release version
+make dist
 ```
 
-Use the `--verbosity=DEBUG` command-line argument for more details.
-
-If you’d like to receive Slack notifications whenever a new release is published, follow the comments in the [Release Notification](https://github.com/jenstroeger/python-package-template/tree/main/.github/workflows/release-notifications.yaml) Action and set up a Slack bot by following [the instructions here](https://github.com/slackapi/slack-github-action#setup-2).
+This builds a source package and a binary distribution, and stores the files in your local `dist/` folder.
 
-In order to build a distribution of your package locally instead of publishing it through the Github Action, you can simply call:
+You can also generate a changelog and bump the version manually and locally using commitizen (already installed as a dev dependency), for example:
 
 ```bash
-make dist
+cz changelog
+cz bump
 ```
 
-This builds a source package and a binary distribution, and stores the files in your local `dist/` folder.
+## Build integrity using SLSA framework
+
+The build process in this repository follows the requirements in the [SLSA framework](https://slsa.dev/) to be compliant at level 3. An important aspect of SLSA to improve the supply chain security posture is to generate a verifiable provenance for the build pipeline. Such a provenance can be used to verify the builder and let the consumers check the materials and configurations used while building an artifact. In this repository we use the [generic provenance generator reusable workflow](https://github.com/slsa-framework/slsa-github-generator) to generate a provenance that can attest to the following artifacts in every release:
+
+* Binary dist (wheel)
+* Source dist (tarball)
+* SBOM (CycloneDx format)
+* HTML Docs
+
+To verify the artifact using the provenance follow the instructions in the [SLSA verifier](https://github.com/slsa-framework/slsa-verifier) project to install the verifier tool. After downloading the artifacts and provenance, verify each artifact individually, e.g.,:
+
+```bash
+slsa-verifier -artifact-path  ~/Downloads/package-2.2.0.tar.gz -provenance attestation.intoto.jsonl -source github.com/jenstroeger/python-package-template
```

Comment:
Maybe remove the `~/Downloads/` because that’s platform dependent?

Reply:
That's just an example to show they have to download it.